README.md

```diff
@@ -1,71 +1,114 @@
 
-Calling out known issues can help limit users opening duplicate issues against your extension.
+```sh
+code --install-extension debricked.vscode-extension
+```
 
-## Release Notes
+## Usage
+After installing the extension, you can start using it by:
 
-Users appreciate release notes as you update your extension.
+1. Opening any project in VS Code.
+2. The extension will automatically scan your project and provide dependency and security analysis in the VS Code interface.
+3. Access detailed reports and recommendations from the Debricked panel in the Activity Bar.
 
-### 1.0.0
+## Development
+### Prerequisites
+Ensure you have the following tools installed:
 
-Initial release of ...
+- Node.js (version 14 or later)
+- npm
+- Visual Studio Code
 
-### 1.0.1
+### Setup
+Clone the repository and install dependencies:
 
-Fixed issue #.
+```sh
+git clone https://github.com/debricked/VS-Code-extension.git
+cd VS-Code-extension
+npm install
+```
 
-### 1.1.0
+### Building
+To compile the extension, run:
 
-Added features X, Y, and Z.
+```sh
+npm run compile
+```
 
----
+The compiled files will be located in the dist folder.
 
-## Following extension guidelines
+### Testing
+Run the tests using:
 
-Ensure that you've read through the extensions guidelines and follow the best practices for creating your extension.
+```sh
+npm run test
+```
 
-* [Extension Guidelines](https://code.visualstudio.com/api/references/extension-guidelines)
+Test coverage reports will be generated in the coverage folder.
 
-## Working with Markdown
+## Contributing
+We welcome contributions from the community! To contribute, please follow these steps:
 
-You can author your README using Visual Studio Code. Here are some useful editor keyboard shortcuts:
+1. Fork the repository.
+2. Create a new branch (git checkout -b feature/YourFeature).
+3. Make your changes and commit them (git commit -m 'Add some feature').
+4. Push to the branch (git push origin feature/YourFeature).
+5. Open a pull request.
+6. For detailed guidelines, refer to our Contributing Guide.
 
-* Split the editor (`Cmd+\` on macOS or `Ctrl+\` on Windows and Linux).
-* Toggle preview (`Shift+Cmd+V` on macOS or `Shift+Ctrl+V` on Windows and Linux).
-* Press `Ctrl+Space` (Windows, Linux, macOS) to see a list of Markdown snippets.
+## License
+This project is licensed under the MIT License. See the LICENSE file for details.
 
-## For more information
+## Acknowledgments
+- Special thanks to the Debricked team for their support and contributions.
+- Inspired by the Visual Studio Code Extension Guide.
 
-* [Visual Studio Code's Markdown Support](http://code.visualstudio.com/docs/languages/markdown)
-* [Markdown Syntax Reference](https://help.github.com/articles/markdown-basics/)
+## Contributors
+We appreciate all the Contributors who have helped make this project better.
 
-**Enjoy!**
+## Milestones
+Check out our Milestones to see our project roadmap and upcoming features.
```

Comment:
We don't have any open milestones etc here yet. Maybe skip this part?

Reply:
removed Milestones as part of Update : boilerplate VS Code Extension #2 PR